cpp/cmake/CPM.cmake

```diff
@@ -0,0 +1,19 @@
+set(CPM_DOWNLOAD_VERSION 0.27.3)
```

Comment:
Instead of using a git tag like this, in RMM we used a git commit hash to make sure this couldn't change out from underneath us. We should use the same approach here.